### ISSUE TITLE
[Other] Standardise error output of the @kipper/cli

### ISSUE THREAD

**Issue by Luna-Klatzer** [Kipper-Lang/Kipper#435]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current State & Issue

Improve the error messages and the output of the CLI, as with various errors there are different styles of tracebacks and error messages, meaning the following errors are not consistent with each other:

- Oclif style errors:
  ```bash
  $ kipper unknown
   ›   Error: command unknown not found
  ```
  
  ```bash
  $ kipper run --target=kip --string-code=";"
   ›   Error: Expected --target=kip to be one of: js, ts
   ›   See more help with --help
  ```
- Custom internal Kipper errors:
  ```bash
  $ kipper run
      KipperInvalidInputError: Argument 'file' or flag '-s/--string-code' must be populated. Aborting...
  ```

**Comment by Luna-Klatzer** [comment]
Should be implemented for patch release [`v0.10.3`](https://github.com/Kipper-Lang/Kipper/pull/486).

**Comment by Luna-Klatzer** [comment]
Fixed by #489 and will be released in `v0.10.3` and the next dev version `0.11.0-alpha.1`.

**Comment by Luna-Klatzer** [comment]
To be implemented as well for `0.11.0-alpha.1` ([`#510`](https://github.com/Kipper-Lang/Kipper/pull/510)).

**Comment by Luna-Klatzer** [comment]
Finished issue with merge of #510.
